provisioning/freeipa-users/tasks/main.yml

```diff
@@ -30,7 +30,7 @@
   shell: echo "{{ ipa_admin_password }}" | kinit admin
 
 - name: "Add user to FreeIPA"
-  shell: ipa user-add {{ item.user }} --first {{ item.first }} --last {{ item.last }}
-  with_items: "{{ freeipa_user }}"
+  shell: echo $'{{ ipa_admin_password }}\n{{ ipa_admin_password }}' | ipa user-add {{ item.user }} --first {{ item.first }} --last {{ item.last }} --password; sleep 1;echo $'{{ ipa_admin_password }}\n{{ ipa_admin_password }}\n{{ ipa_admin_password }}'| kinit {{ item.user }};echo $'{{ ipa_admin_password }}'| kinit admin
```

Comment:
this line properly gives every user a new password by logging-in once and setting a 'new' password